### ISSUE TITLE
Reduce the bundle size

### ISSUE THREAD

**Issue by aminya** [idleberg/atom-package-control#1]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I checked the bundle size, and it is 400KB, which is quite high. I was wondering if it is possible to reduce this.
Some of the suggestions:
- Using lightweight replacement for the dependencies.
- Use dynamic imports
- Target a recent Atom version. The latest version of Atom is on Electron 9, which means it is the same as Chrome 81. If you want to support a slightly older Atom, you can target Electron 6 and remove core-js dependency.
- Do not bundle the dependencies that are already cached somewhere else. Atom-select-list is probably one of those.

### Steps to Reproduce

1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Reproduces how often:**

**Comment by idleberg** [comment]
I've addressed a couple of your suggestions in v0.12, however, I did not manage to import `atom-select-list` dynamically. The result was the cancellation of the select list (`didCancelSelection()` was fired).

> If you want to support a slightly older Atom, you can target Electron 6 and remove core-js dependency

This does not any sense to me, maybe you can explain. Isn't it usually the opposite, that targeting a *newer* version allows omitting polyfills?

**Comment by aminya** [comment]
That's awesome! 

Yeah, here I meant using a recent Atom target, which seems you are already doing it

**Comment by idleberg** [comment]
PS: I think I got the dynamic imports working, I didn't import `default` in my previous attempt:

```js
const SelectList = (await import('atom-select-list')).default;
```

**Comment by idleberg** [comment]
I think I addressed all your points in v0.12.2

**Comment by aminya** [comment]
That's great news! I will close this for now.
